.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
+      - run: npm install -force
```

Comment:
Is this definitely needed? I didn't need to use force when installing locally. FIne if it is needed, just curious why I didn't need to run it.

Reply:
It is nedded, @use-debouncy library needs to be migrated to React18.